larpandoracontent/LArTwoDReco/TrackInEMShowerAlgorithm.h

```diff
@@ -1,12 +1,12 @@
 /**
- *  @file   TrackInEMShowerAlgorithm.h
+ *  @file   larpandoracontent/LArTwoDReco/TrackInEMShowerAlgorithm.h
  *
- *  @brief  Header file for the em track algorithm class
+ *  @brief  Header file for the track in em shower algorithm class
  *
  *  $Log: $
  */
-#ifndef LAR_EM_TRACK_ALGORITHM_H
-#define LAR_EM_TRACK_ALGORITHM_H 1
+#ifndef LAR_TRACK_IN_EM_SHOWER_ALGORITHM_H
+#define LAR_TRACK_IN_EM_SHOWER_ALGORITHM_H 1
 
 #include "Pandora/Algorithm.h"
 
```

Comment:
You probably need <unordered_map> here now too, below the Pandora and larpandoracontent includes.